docs/source/data.rst

```diff
@@ -164,6 +164,37 @@ Data
 `b3-2016-06-07-12-42-49.bag <https://storage.googleapis.com/cartographer-public-data/bags/backpack_3d/b3-2016-06-07-12-42-49.bag>`_                  596 s    3.9 GB 3 gaps in horizontal laser data, no intensities
 ==================================================================================================================================================== ======== ====== ================================================
 
+MiR
+===========================================
+
+This data was collected using `MiR100 <http://www.mobile-industrial-robots.com/de/products/mir100/>`_.
```

Comment:
maybe mention that we attached an additional camera (RealSense?) on top of the robot.

Reply:
done